contracts/VToken.sol

```diff
@@ -807,9 +1061,30 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
         uint256 accountBorrowsNew = accountBorrowsPrev - actualRepayAmount;
         uint256 totalBorrowsNew = totalBorrows - actualRepayAmount;
 
-        /* We write the previously calculated values into storage */
-        accountBorrows[borrower].principal = accountBorrowsNew;
-        accountBorrows[borrower].interestIndex = borrowIndex;
+        if (InterestRateMode(interestRateMode) == InterestRateMode.STABLE) {
+            uint256 stableBorrowsNew = stableBorrows - actualRepayAmount;
+
+            uint256 stableRateMantissa = accountStableBorrows[borrower].stableRateMantissa;
```

Comment:
We could define this variable only in the `else` statement, where it is used, to save some gas

Reply:
Fixed, can verify the changes at [commit](https://github.com/VenusProtocol/isolated-pools/pull/132/commits/0f14951544955613711dd9321a85af4f96073049).